CHANGES.txt

```diff
@@ -20,6 +20,11 @@ New features:
 - Use of old style range tuples as windows is deprecated and warned
   against (#1074).
 
+Bug fixes:
+
+- Eliminate boundless keyword argument and deprecation warnings when calling
+  from_bounds() (#1082).
```

Comment:
Is it true that the windows created from bounds are always boundless by design? Should we highlight that, maybe as a "New Feature" to explain why the kwarg is deprecated?